src/classes/LD_LeadConvertOverride_TEST.cls

```diff
@@ -91,15 +91,19 @@ private class LD_LeadConvertOverride_TEST {
         ext.getConList();
         system.assertEquals(2, ext.conList.size());
         system.assert(ext.lcStatuses.size() > 0);
-        system.assert(ext.leadConvStatus != null && ext.leadConvStatus != '');
         system.assertEquals(1, ext.listSOAccounts.size());
         system.assertEquals(1, ext.listSOAfflAccounts.size());
 
         // this selects the "New" option in the Contact box
         ext.strConId = 'NEW_CONTACT';
         ext.strAccountId = 'AUTO';
+
         Test.startTest();
-        ext.convertLeadNow();
+        for (SelectOption lcStatus : ext.lcStatuses) {
```

Comment:
I cannot find a way since the page never really loads in the test, it doesn't know the valid values based on Visualforce filtering by recordType for us. The only way I could find to get this information in Apex was by using that metadata api workaround. I'm also working under the assumption that there won't be too many orgs set up with too many converted statuses based on record types.